topostats/utils.py

```diff
@@ -269,14 +296,14 @@ def get_thresholds(  # noqa: C901
     return thresholds
 
 
-def create_empty_dataframe(columns: set = ALL_STATISTICS_COLUMNS, index: str = "grain_number") -> pd.DataFrame:
+def create_empty_dataframe(column_set: str = "grainstats", index: str = "grain_number") -> pd.DataFrame:
```

Comment:
I realised `index` is a common Python method and will change this to `index_col` to avoid any possible confusion in the work I'm doing to parametrise the tests.